index.html

```diff
@@ -224,35 +238,37 @@ <h2>
       </p>
       <ol class="postures">
         <li>
-          <img src="images/nofold-mode-graphic.svg" class="drawing" alt="drawing of no-fold mode">
-          <a>No-Fold</a> is the posture of a device without a hinge. 
-          This is the expected value for devices that do not fold.
-        </li>
-        <li>
-          <img src="images/laptop-mode-graphic.svg" class="drawing" alt="drawing of laptop mode">
-          <a>Laptop</a> posture indicates that the device is being used as
-          a traditional laptop, meaning one screen is placed on a more or
-          less horizontal surface with a screen angle between 180 to 0 degrees.
-        </li>
-        <li>
-          <img src="images/flat-mode-graphic.svg" class="drawing" alt="drawing of flat mode">
-          <a>Flat</a> posture indicates that one screen is being placed on
-          a more or less horizontal surface with a screen angle around 180 degrees.
+          <img src="images/continuos-posture.svg" class="drawing" alt="drawing of the continuous posture">
+          <p><a>Continuous</a> posture: The continuous posture refers to a "flat" position. This is the default case.</p>
```

Comment:
to a flat, folded out position. This is the default case for most devices not allowing different postures